pkg/middleware/middleware.go

```diff
@@ -59,8 +60,22 @@ func Process(ctx *quaycontext.QuayRegistryContext, quay *v1.QuayRegistry, obj cl
 			return dep, nil
 		}
 
+		// Set env variable overrides
+		var envVars map[string]string
+		switch quayComponentLabel {
+		case "postgres":
+			envVars = getEnvOverrideForComponent(quay, v1.ComponentPostgres)
+		case "clair-postgres":
+			envVars = getEnvOverrideForComponent(quay, v1.ComponentClair)
+		case "objectstorage":
```

Comment:
What deployment contains as its component label "objectstorage" ? This does not seem right. where are we setting the override for the object storage?